raiden_contracts/tests/test_channel_cooperative_settle.py

```diff
@@ -10,7 +11,7 @@
 from raiden_contracts.utils.events import check_channel_settled, check_withdraw_2
 from raiden_contracts.utils.type_aliases import BlockExpiration
 
-EXPIRATION = BlockExpiration(100)
+EXPIRATION = int(time.time()) + BlockExpiration(100) * 15
```

Comment:
Shouldn't this just be an relative value as before?

Reply:
This was being used in an absolute way (using the 100th block as the expiration block), so we have to calculate the timestamp of the 100th block to keep things as they were.